### DISCUSSION TITLE
Urban Heat Island: Urban Weather Generator error

### DISCUSSION THREAD

**Discussion by SIGGEO99** [UMEP-dev/UMEP#649] [Q&A]
hi guys
Can you help me with this error when I run the urban heat island: urban weather generator.

![umep](https://github.com/user-attachments/assets/107c2ea6-9008-4aaa-8c01-0364e064d41e)

**Comment by biglimp** [comment]
Try running the model again and have the python console open while executing, to see a more detailed output from the model.

**Reply by SIGGEO99** [reply]
Hi, thanks for help.
I try with python console open, and this appears.
Can you explain me what this error means and how can I solve it?
![python](https://github.com/user-attachments/assets/3151564a-cf82-4ac8-beac-2c916d6dfbef)

**Reply by biglimp** [reply]
Looks like your latitude might be in wrong format. Please attach one of your .uwg-files.

**Reply by SIGGEO99** [reply]
Thanks for help. It's here one of my .uwg-files 

[uwg.zip](https://github.com/user-attachments/files/17280213/uwg.zip)

**Reply by biglimp** [reply]
Your .epw-file is in wrong format. UWG require an energy plus weather file, not a regular umep metfile.

**Reply by SIGGEO99** [reply]
Ok, do you know how I can solve this problem?

**Reply by biglimp** [reply]
You can either convert your data to .epw by adding header information (make sure that you have a full year of data, hourly resolution) or download an .epw-file from your area. There are different places to this e.g. www.shinyweatherdata.com or https://energyplus.net/weather

**Reply by SIGGEO99** [reply]
Thanks for your help.
I will try that

**Reply by SIGGEO99** [reply]
Hi, I already managed to solve the previous error. Now in the post-processor phase it gives this error.
How can I resolve it?

![erro3](https://github.com/user-attachments/assets/f8781f98-a478-4edf-a4c7-3bad1fe4640e)

**Reply by biglimp** [reply]
See this closed issue: https://github.com/UMEP-dev/UMEP/issues/648
